dask_sql/context.py

```diff
@@ -274,8 +274,10 @@ def register_function(
             SELECT f(x)
             FROM df
 
-        Please note that you can always only have one function with the same name;
-        no matter if it is an aggregation or scalar function.
+        Please keep in mind that you can only have one function with the same name,
+        regardless of whether it is an aggregation or a scalar function. However,
+        if you register two functions with the same name, the last one registered
+        will take precedence over the first one.
```

Comment:
Should we specify here that the precedence only applies if `replace=True`?

```
Please keep in mind that you can only have one function with the same name,
regardless of whether it is an aggregation or a scalar function. By default,
attempting to register two functions with the same name will raise an error;
setting `replace=True` will give precedence to the most recently registered
function.
```

Reply:
> Should we specify here that the precedence only applies if `replace=True`?
> 
> ```
> Please keep in mind that you can only have one function with the same name,
> regardless of whether it is an aggregation or a scalar function. By default,
> attempting to register two functions with the same name will raise an error;
> setting `replace=True` will give precedence to the most recently registered
> function.
> ```

Yes, we should; thanks for catching that!